brain_observatory_analysis/utilities/image_utils.py

```diff
@@ -136,3 +136,22 @@ def save_figure(fig, figsize, save_dir, folder, fig_title, formats=['.png', '.pd
     fig.set_size_inches(figsize)
     for f in formats:
         fig.savefig(filename + f, transparent=True, orientation='landscape', bbox_inches='tight', dpi=300, facecolor=fig.get_facecolor())
+
+def generate_distinct_colors(num_colors):
+    """Generate distinct colors
+    Parameters
+    ----------
+    num_colors : int
+        Number of colors to generate
+    Returns
+    -------
+    list
+        List of RGB tuples
+    """
+
+    colors = []
+    for i in range(num_colors):
+        hue = i / num_colors
```

Comment:
This does not reach 1 - is it intended?
And is this about grayscale or colors? (hsv input only has hue, no saturation and value)

Reply:
yes, this is because 1 would be white which is not visible, so it goes from light grey to black.